### DISCUSSION TITLE
I have a smart plugs with power monitoring that randomly stops working

### DISCUSSION THREAD

**Discussion by olavgg** [jasonacox/tinytuya#462] [Q&A]
I have 4x tuya smart plugs which runs v3.4 and 4x other smart plugs that runs v3.3.  Those smart plugs that runs v3.4 will between a few minutes to a few hours stop responding and return: 
`status: {'Error': 'Check device key or version', 'Err': '914', 'Payload': None}`

To get the device working again I need to remove it from Smart Life and re-add it with a new key.

Have anyone else had a similiar issue and know about a workaround?
The smart plugs running v3.3 works fine. No issues.

**Comment by vjbutt** [comment]
I also have the same issue with an energy monitoring device running v3.4.  I can still access the device using the Smart Life app, but tinytuya gives the same error as yours.

Unfortunately the only solution that I have found so far is to power cycle.

**Reply by JAPISOFT** [reply]
The same problem!

**Comment by uzlonewolf** [comment]
> To get the device working again I need to remove it from Smart Life and re-add it with a new key.

I can't say I've run across any devices that require this, but most of my v3.4 devices do need to be power cycled every now and then.  I don't know for sure but I suspect memory corruption is causing it to temporarily "forget" what the local key is supposed to be.
